### DISCUSSION TITLE
Show Ghommal's Hilt Teleport Location on the World Map

### DISCUSSION THREAD

**Discussion by DrYoshiyahu** [runelite/runelite#16005] [Q&A] [answered]
Pretty self-explanatory. Ghommal's Hilt has a teleport just like many of the Achievement Diary rewards, which is located north of Trollheim, about here. Much like the Achievement Diary teleports, it would be nice to have this location shown on the map.

![image](https://user-images.githubusercontent.com/14756430/206188897-41987c79-949e-410b-9f23-5b2a4d5e52c6.png)

**Comment by Nightfirecat** [comment, marked as answer]
I've converted this to an issue for tracking: #16006
